packages/kit/src/components/AccountSelector/NetworkSelectorTrigger/NetworkSelectorTrigger.tsx

```
@@ -99,69 +98,53 @@ function NetworkSelectorTriggerHomeCmp({ num }: { num: number }) {
   );
 
-  const trigger = useMemo(
-    () => (
-      <XStack
-        testID="account-network-trigger-button"
-        role="button"
+  return (
+    <XStack
+      testID="account-network-trigger-button"
+      role="button"
+      flexShrink={1}
```

Biome

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)